src/ServerConfig.cc

```diff
@@ -915,59 +915,59 @@ ignition::gazebo::loadPluginInfo(bool _isPlayback)
   }
 
   std::string defaultConfigDir;
-  ignition::common::env(IGN_HOMEDIR, defaultConfigDir);
-  defaultConfigDir = ignition::common::joinPaths(defaultConfigDir, ".ignition",
-    "gazebo", IGNITION_GAZEBO_MAJOR_VERSION_STR);
+  gz::common::env(IGN_HOMEDIR, defaultConfigDir);
+  defaultConfigDir = gz::common::joinPaths(defaultConfigDir, ".gz",
+    "sim", GZ_SIM_MAJOR_VERSION_STR);
 
-  auto defaultConfig = ignition::common::joinPaths(defaultConfigDir,
+  auto defaultConfig = gz::common::joinPaths(defaultConfigDir,
       configFilename);
 
-  if (!ignition::common::exists(defaultConfig))
+  if (!gz::common::exists(defaultConfig))
   {
-    auto installedConfig = ignition::common::joinPaths(
-        IGNITION_GAZEBO_SERVER_CONFIG_PATH,
+    auto installedConfig = gz::common::joinPaths(
+        IGN_GAZEBO_SERVER_CONFIG_PATH,
```

Comment:
I needed to change this to compile

```suggestion
        GZ_SIM_SERVER_CONFIG_PATH,,
```